org.netsurf_browser.NetSurf.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 NetSurf developers -->
+<component type="desktop-application">
+  <id>org.netsurf_browser.NetSurf</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  <project_license>MIT</project_license>
+  <name>NetSurf</name>
+  <summary>small web browser with CSS support</summary>
+  <developer_name>NetSurf Developers</developer_name>
+
+  <description>
+    <p>
+     NetSurf is a multi-platform lightweight web browser. It aims to provide
+     comprehensive rendering of HTML 5 with CSS 2 in a small resource footprint
+     without compromising performance.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">org.netsurf_browser.NetSurf.desktop</launchable>
+
+  <url type="bugtracker">https://bugs.netsurf-browser.org/</url>
+  <url type="help">https://www.netsurf-browser.org/documentation/</url>
+  <url type="homepage">https://www.netsurf-browser.org/></url>
+
+  <releases>
+    <release version="3.10" date="2020-05-24">
+      <description>
+        <ul>
+          <li>Stable release</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+
+  <content_rating type="oars-1.1">
```

Comment:
It can be shortened to `<content_rating type="oars-1.1" />`.

Reply:
Thankyou, I have made that change